Permanent/Views/Cells/ArchiveTableViewCell/ArchiveScreenSectionTitleTableViewCell.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  ArchiveScreenTopSectionTableViewCell.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 15.09.2021.
+//
+
+import UIKit
+
+class ArchiveScreenSectionTitleTableViewCell: UITableViewCell {
```

Comment:
this should be a subclass of UITableViewHeaderFooterView

Reply:
Changed UITableViewCell with UITableViewHeaderFooterView.